_posts/2018-10-22-homebrew-1.8.0.md

```diff
@@ -0,0 +1,43 @@
+---
+title: 1.8.0
+author: MikeMcQuaid
+---
+Today I'd like to announce Homebrew 1.8.0. The most significant changes since 1.8.0 are .
```

Comment:
```suggestion
Today I'd like to announce Homebrew 1.8.0. The most significant changes since 1.7.0 are official Mojave support, linkage auto-repair on `brew upgrade` and quarantining Cask’s downloads.
```